cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -1591,35 +1620,35 @@ subroutine div_stress  (nx_block,   ny_block,   &
-                                      -(dxT_U(i,j-1)**2)*stressm(i,j-1) )  &
-                 + (c1/dyE_N(i,j)) * ( (dyT_U(i+1,j)**2) * stress12(i+1,j) &
-                                      -(dyT_U(i,j)**2)*stress12(i,j) ) )
+            F1(i,j) = (c1/area(i,j)) * &
+                 ( p5 * dyE_N(i,j) * ( stresspF1(i+1,j)-stresspF1(i,j) )     &
+                 + (p5/dyE_N(i,j)) * ( (dyT_U(i+1,j)**2) * stressmF1(i+1,j)  &
+                                      -(dyT_U(i,j)**2)*stressmF1(i,j) )      &
+                 + (c1/dxE_N(i,j)) * ( (dxT_U(i,j)**2) * stress12F1(i,j)     &
+                                      -(dxT_U(i,j-1)**2)*stress12F1(i,j-1) ) )
+
+            F2(i,j) = (c1/area(i,j)) * &
+                 ( p5 * dxE_N(i,j) * ( stresspF2(i,j)-stresspF2(i,j-1) )     &
+                 - (p5/dxE_N(i,j)) * ( (dxT_U(i,j)**2) * stressmF2(i,j)      &
+                                      -(dxT_U(i,j-1)**2)*stressmF2(i,j-1) )  &
+                 + (c1/dyE_N(i,j)) * ( (dyT_U(i+1,j)**2) * stress12F2(i+1,j) &
+                                      -(dyT_U(i,j)**2)*stress12F2(i,j) ) )
 
          case('N')
 
-            F1(i,j) = arear(i,j) * &
-                 ( p5 * dyE_N(i,j) * ( stressp(i,j)-stressp(i-1,j) )       &
-                 + (p5/dyE_N(i,j)) * ( (dyT_U(i,j)**2) * stressm(i,j)      &
-                                      -(dyT_U(i-1,j)**2)*stressm(i-1,j) )  &
-                 + (c1/dxE_N(i,j)) * ( (dxT_U(i,j+1)**2) * stress12(i,j+1) &
-                                      -(dxT_U(i,j)**2)*stress12(i,j) ) )
-
-            F2(i,j) = arear(i,j) * &
-                 ( p5 * dxE_N(i,j) * ( stressp(i,j+1)-stressp(i,j) )       &
-                 - (p5/dxE_N(i,j)) * ( (dxT_U(i,j+1)**2) * stressm(i,j+1)  &
-                                      -(dxT_U(i,j)**2)*stressm(i,j) )      &
-                 + (c1/dyE_N(i,j)) * ( (dyT_U(i,j)**2) * stress12(i,j)     &
-                                      -(dyT_U(i-1,j)**2)*stress12(i-1,j) ) )
+            F1(i,j) = (c1/area(i,j)) * &
+                 ( p5 * dyE_N(i,j) * ( stresspF1(i,j)-stresspF1(i-1,j) )     &
+                 + (p5/dyE_N(i,j)) * ( (dyT_U(i,j)**2) * stressmF1(i,j)      &
+                                      -(dyT_U(i-1,j)**2)*stressmF1(i-1,j) )  &
+                 + (c1/dxE_N(i,j)) * ( (dxT_U(i,j+1)**2) * stress12F1(i,j+1) &
+                                      -(dxT_U(i,j)**2)*stress12F1(i,j) ) )
+
+            F2(i,j) = (c1/area(i,j)) * &
+                 ( p5 * dxE_N(i,j) * ( stresspF2(i,j+1)-stresspF2(i,j) )     &
+                 - (p5/dxE_N(i,j)) * ( (dxT_U(i,j+1)**2) * stressmF2(i,j+1)  &
+                                      -(dxT_U(i,j)**2)*stressmF2(i,j) )      &
+                 + (c1/dyE_N(i,j)) * ( (dyT_U(i,j)**2) * stress12F2(i,j)     &
+                                      -(dyT_U(i-1,j)**2)*stress12F2(i-1,j) ) )
          case default
             call abort_ice(subname // ' unkwown grid_location: ' // grid_location)
```

Comment:
spell unknown

Reply:
I fixed the spelling here and elsewhere in #36